packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -141,11 +141,16 @@ const Redirect = observer(() => {
         }
         case 'payment_withdraw': {
             if (is_next_wallet) {
-                // passes verification_code through query param as we do not want to use localstorage/session storage
-                // though can't use "verification_code" as name param
-                // as there is general logic within client-store
-                // which removes anything which resembles code=XYZ
-                history.push(`${routes.wallets_withdrawal}?verification=${verification_code?.payment_withdraw}`);
+                // 1. pass verification_code through query param as we do not want to use localstorage/session storage
```

Comment:
just a small nit-pick: can we use multiline comment syntax as we have multiple lines here? `/* */`
 The rest LGTM :)